l10n_de_country_states/models/res_country.py

```diff
@@ -1,29 +1,9 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-# OpenERP, Open Source Management Solution
-# Copyright (C) 2010-2013 OpenERP s.a. (<http://openerp.com>).
-# Copyright (C) 2014 initOS GmbH & Co. KG (<http://www.initos.com>).
-# Author Nikolina Todorova <nikolina.todorova@initos.com>
-#        (c) 2015 Antiun Ingenieria, SL (Madrid, Spain, http://www.antiun.com)
-#                 Antonio Espinosa <antonioea@antiun.com>
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero General Public License as
-# published by the Free Software Foundation, either version 3 of the
-# License, or (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-# GNU Affero General Public License for more details.
-#
-# You should have received a copy of the GNU Affero General Public License
-# along with this program. If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2015 Nikolina Todorova <nikolina.todorova@initos.com>
+# Copyright 2015 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from openerp import fields, models
+from odoo import fields, models
 
 
 class CountryState(models.Model):
```

Comment:
@pedrobaeza Agree. Maybe we find some time in October to discuss the "optimal" relation between generalizability and specificity, since that is at least to my point of view something that is comparably poorly handled within the Odoo ecosystem (and many other OS frameworks). Sorry for kind of offtopic